tests/php/CapabilitiesTest.php

```diff
@@ -174,6 +185,15 @@ public function testGetCapabilitiesGuest(): void {
 						'enabled' => true,
 						'breakout-rooms' => false,
 						'recording' => false,
+						'predefined-backgrounds' => [
```

Comment:
Didn't we say we were going to loop over the files in the folders and get the names from there instead of hard-coding them?

Reply:
This is the test and just making sure that the iteration is working as expected :D